pyne/alara.py

```diff
@@ -296,19 +297,34 @@ def record_to_geom(mesh, cell_fracs, cell_mats, geom_file, matlib_file,
 
     printed_mats = []
     for mat in cell_mats.values():
-        mat_num = mat.metadata['mat_number']
-        if mat_num not in printed_mats:
-            printed_mats.append(mat_num)
-            matlib += "mat_{0}    {1: 1.6E}    {2}\n".format(
-                       mat.metadata['mat_number'], mat.density, len(mat.comp))
+        name = mat.metadata['name']
+        if _is_void(name):
+            print_void = True
+            continue
+        if name not in printed_mats:
+            printed_mats.append(name)
+            matlib += "{0}    {1: 1.6E}    {2}\n".format(name, mat.density,
+                                                         len(mat.comp))
             for nuc, comp in mat.comp.iteritems():
                 matlib += "{0}    {1: 1.6E}    {2}\n".format(alara(nuc), 
                                                       comp*100.0, znum(nuc))
             matlib += "\n"
 
+    if print_void:
+       matlib += "# void material\nmat_void 0.0 1\nhe 1 2\n"
+
     with open(matlib_file, 'w') as f:
         f.write(matlib)
 
+def _is_void(name):
+    """Private function for determining if a material name specifies void.
+    """
+    if "vacuum" in name.lower() or "void" in name.lower() \
+        or "graveyard" in name.lower():
+        return True
+    else:
+       return False
```

Comment:
not that this is blocking either but this could be rewritten as just 

```python
return "vacuum" in name.lower() or "void" in name.lower() or "graveyard" in name.lower()
```

Or the faster:

```python
lname = name.lower()
return "vacuum" in lname or "void" in lname or "graveyard" in lname
```